bee-tangle/src/traversal.rs

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! Collection of Tangle traversal functions.
+
+// TODO: Refactor all of this into methods on `Tangle`.
```

Comment:
Is this `TODO` intended for this PR?

Reply:
It is not. It's still relevant, but not something to resolve now.